libraries/AP_Proximity/AP_Proximity.cpp

```diff
@@ -229,7 +229,7 @@ void AP_Proximity::init()
 #if AP_PROXIMITY_MR72_ENABLED
         case Type::MR72:
             state[instance].instance = instance;
-            drivers[instance] = NEW_NOTHROW AP_Proximity_MR72_CAN(*this, state[instance], params[instance]);
+            drivers[instance] = NEW_NOTHROW AP_Proximity_MR72_CAN(*this, state[instance], params[instance], AP_Proximity_MR72_CAN::Manufacturer::NanoRadar);
```

Comment:
Why not just pass the protocol name directly here?

Reply:
We share the CAN protocol now so the argument in question hasn't been added any more